libraries/anvill_passes/src/TransformRemillJumpIntrinsics.cpp

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2021 Trail of Bits, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <anvill/Analysis/CrossReferenceResolver.h>
+#include <anvill/Analysis/Utils.h>
+#include <anvill/Lifters/EntityLifter.h>
+#include <anvill/Transforms.h>
+#include <glog/logging.h>
+#include <llvm/ADT/Triple.h>
+#include <llvm/IR/Function.h>
+#include <llvm/IR/InstIterator.h>
+#include <llvm/IR/Instruction.h>
+#include <llvm/IR/Intrinsics.h>
+#include <llvm/IR/Module.h>
+#include <llvm/Pass.h>
+#include <remill/BC/Util.h>
+
+#include <utility>
+#include <vector>
+
+
+namespace anvill {
+namespace {
+
+const std::string_view remill_jump = "__remill_jump";
```

Comment:
Make these into `llvm::StringRef` so that below you don't need to `== remill_jump.data()`